project/AppDependencies.scala

```diff
@@ -6,10 +6,10 @@ object AppDependencies {
 
   private val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"                %% "bootstrap-frontend-play-30" % bootstrapVersion,
-    "uk.gov.hmrc"                %% "play-frontend-hmrc-play-30" % "10.9.0",
+    "uk.gov.hmrc"                %% "play-frontend-hmrc-play-30" % "10.10.0",
     "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-30"         % "2.2.0",
     "com.beachape"               %% "enumeratum-play"            % "1.8.1",
-    "org.webjars.npm"             % "accessible-autocomplete"    % "2.0.4",
+    "org.webjars.npm"             % "accessible-autocomplete"    % "3.0.0",
```

Comment:
Since this is a major version, I recommend not updating unless it has been thoroughly tested, as there are breaking changes, as detailed [here](https://github.com/alphagov/accessible-autocomplete/releases/tag/v3.0.0).